src/guide/transitions-state.md

```diff
@@ -161,13 +84,22 @@ Managing many state transitions can quickly increase the complexity of a Vue ins
 ```
 
 ```js
-// This complex tweening logic can now be reused between
-// any integers we may wish to animate in our application.
-// Components also offer a clean interface for configuring
-// more dynamic transitions and complex transition
-// strategies.
-Vue.component('animated-integer', {
-  template: '<span>{{ tweeningValue }}</span>',
+const app = Vue.createApp({
+  data() {
+    return {
+      firstNumber: 20,
+      secondNumber: 40
+    }
+  },
+  computed: {
+    result() {
+      return this.firstNumber + this.secondNumber
+    }
+  }
+})
+
+app.component('animated-integer', {
+  template: '<span>{{ fullVal }}</span>',
```

Comment:
```suggestion
  template: '<span>{{ fullValue }}</span>',
```

Reply:
Great point! I'll update this in the codepen as well